slack-api-client/src/main/java/com/slack/api/scim/model/User.java

```diff
@@ -37,30 +44,44 @@
 
     @SerializedName(Schemas.SCHEMA_EXTENSION_ENTERPRISE)
     private Extension extension;
+    @SerializedName(Schemas.SCHEMA_EXTENSION_SLACK_GUEST)
+    private SlackGuest slackGuest;
 
     private List<Group> groups;
 
     @Data
+    @Builder
```

Comment:
These annotations generate builder interface for more easily constructing an instance. Refer to the test code building `SlackGuest` instances.